org.kde.Kid3.yaml

```diff
@@ -0,0 +1,78 @@
+app-id: org.kde.Kid3
+runtime: org.kde.Platform
+runtime-version: "5.14"
+sdk: org.kde.Sdk
+command: kid3
+
+rename-desktop-file: org.kde.kid3.desktop
+rename-appdata-file: org.kde.kid3.appdata.xml
+rename-icon: kid3-qt
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=session-bus
```

Comment:
Why it needs session-bus access?

Reply:
I thought that this is necessary to start the application when the session is restored, but this is not the case, so it can be removed.